rust/tools/authorization-logic/BUILD

```diff
@@ -112,6 +112,8 @@ rust_library(
     crate_type = "staticlib",
     rustc_flags = [
         "--cfg=feature=\"bazel_build\"",
+        "-C",
+        "target-feature=+crt-static",
```

Comment:
It looks like static linking is a very platform specific option in Rust. Is it wise to be forcing that to occur? See https://rust-lang.github.io/rfcs/1721-crt-static.html .